firestore/README.md

```diff
@@ -109,6 +113,18 @@ updates with the `EventListener` on the `Query`.
 
 Fear not, FirebaseUI does all of this for you automatically!
 
+
+### Choosing an adapter
+
+FirebaseUI offers two types of RecyclerView adapters for Cloud Firestore:
+
+  * `FirestoreRecyclerAdapter` - bind a `Query` to a `RecyclerView` and respond to all real-time
```

Comment:
I fancy using `—` instead of `-` 😁. Also, should we change the tense to `binds [...] responds ...`?